index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>AllIsCounted</title>
+</head>
+<body>
+<p id="count-clicks">0</p>
```

Comment:
remove this `0`. Try to put default value using js

Reply:
Oops, i forgot to telete this 0. My script puts default value and this 0 doesn`t matter.